blocks/subscriptions-block/features/checkout/_children/BillingAddress/index.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import { usePhrases, Input, Button, Stack } from "@wpmedia/arc-themes-components";
+import countryCodes from "../../../../components/ContactInfo/countryCodes";
+import listOfStates from "./listOfStates";
+
+export const ARCXP_BILLING_ADDRESS = "ArcXP_billingAddress"
+
+const BillingAddress = ({ Sales, user, setError, setOrder, captchaToken, billingAddress, setBillingAddress, className, setIsOpen, setIsComplete, resetRecaptcha, setResetRecaptcha, children }) => {
+	const formRef = useRef();
+	const entriesRef = useRef({});
+  const [isUS, setIsUS] = useState(false);
```

Comment:
Hi Ed, do you mind change this to "isCountryUS", I think it's more self-descriptive